src/core/config/descriptions.h

```diff
@@ -77,4 +77,13 @@ constexpr auto kDWeight =
 constexpr auto kDBumpsLimit = "max considered intervals amount. Pass 0 to remove limit";
 constexpr auto kDIterationsLimit = "limit for iterations of sampling";
 constexpr auto kDACSeed = "seed, needed for choosing a data sample";
+constexpr auto kDHllAccuracy = "(Faida special) HyperLogLog approximation accuracy";
+constexpr auto kDSampleSize = "(Faida special) Size of a table sample";
+constexpr auto kDFindNary = "(Faida special) Detect n-ary inclusion dependencies [true|false]";
+constexpr auto kDIgnoreNullCols =
+        "(Faida special) Ignore INDs which contain columns filled only with NULLs. May increase "
+        "performance but impacts the result. [true|false]";
+constexpr auto kDIgnoreConstantCols =
+        "(Faida special) Ignore INDs which contain columns filled with only one value. May "
+        "increase performance but impacts the result. [true|false]";
```

Comment:
You definitely need to remove these postscripts at the beginning. From python `cli` we can get a set of its options for each specific algorithm. And these inscriptions `(Faida special)` will look strange.